src/app/models/reservation.js

```diff
@@ -0,0 +1,31 @@
+import Backbone from 'backbone';
+
+//how to subclass one of the Backbone classes
+const Reservation = Backbone.Model.extend({
+  urlRoot: function(){
+    return 'https://ada-backtrek-api.herokuapp.com/trips/' + this.attributes.tripId + '/reservations';
```

Comment:
It's preferable to use string interpolation instead of concatenation (as a matter of style).

```
return `https://ada-backtrek-api.herokuapp.com/trips/${this.attributes.tripId}/reservations`;
```